_posts/2023-02-08-zenml-templates.md

```diff
@@ -0,0 +1,83 @@
+---
+layout: post
+author: "Alex Strick van Linschoten"
+title: "Build ML Models Faster with ZenML Project Templates"
+description: "Getting started with your ML project work is easier than ever with Project Templates, a new way to generate scaffolding and a skeleton project structure based on best practices."
+category: zenml
+tags: tooling zenml evergreen
+publish_date: February 8, 2023
+date: 2023-02-08T00:02:00Z
+thumbnail: /assets/posts/templates/project-thumb.png
+image:
+  path: /assets/posts/templates/project-thumb.png
+---
+
+**Last updated:** February 8, 2023
+
+Machine learning projects can be time-consuming and complex, with many tasks that must be completed before model training can begin. From data preprocessing and feature engineering to model selection and deployment, there are many steps involved in a typical ML project. Not only do you have to keep the problem domain in your head, but industry best practices suggest you do certain things — versioning, modularisation, linting, and so on — that can be fiddly to set up. All of this can make it difficult to quickly get started with a new project and can lead to frustration and wasted time. The good news is that there are tools available that can help streamline the process, and one of these is [ZenML Project Templates](https://github.com/zenml-io/zenml-project-templates#readme). These templates provide a starting point for new projects, with a well-structured codebase, commonly-used libraries, and example code to help you hit the ground running.
```

Comment:
```suggestion
Machine learning projects can be time-consuming and complex, with many tasks that must be completed before model training can begin. From data preprocessing and feature engineering to model selection and deployment, there are many steps involved in a typical ML project. Not only do you have to keep the problem domain in your head, but industry best practices suggest you do certain things — versioning, modularization, linting, and so on — that can be fiddly to set up. All of this can make it difficult to quickly get started with a new project and can lead to frustration and wasted time. The good news is that there are tools available that can help streamline the process, and one of these is [ZenML Project Templates](https://github.com/zenml-io/zenml-project-templates#readme). These templates provide a starting point for new projects, with a well-structured codebase, commonly-used libraries, and example code to help you hit the ground running.
```